google-cloud-pubsub/lib/google/cloud/pubsub/topic.rb

```diff
@@ -1085,13 +1092,6 @@ def ensure_grpc!
           ensure_service!
           reload! if reference?
         end
-
-        ##
-        # Call the publish API with arrays of data data and attrs.
-        def publish_batch_messages batch
```

Comment:
Is this the same method which you moved to `batch_publisher` ?

Reply:
This method is moved to publisher as it is the functionality of the class and it was evident as we were using most of the values from the passed in batch instance.